Integrations/MicrosoftManagementActivity/README.md

```diff
@@ -0,0 +1,17 @@
+Microsoft Management Activity API (O365/Azure Events) should be used to retrieve content records from the various Microsoft Management Activity content types.
```

Comment:
Please update the readme to include the docs for the integration itself too once Alex will complete is review